gnat2goto/driver/tree_walk.adb

```diff
@@ -2921,7 +2921,7 @@ package body Tree_Walk is
       Set_Identifier (Proc, To_String (Callee));
 
       Set_Source_Location (R, Sloc (N));
-      --  Set_LHS (R, Empty);  -- ??? what is the "nil" irep?
+      --  Set_LHS (R, Make_Nil (Sloc (N)));
```

Comment:
Should this be no longer commented now?

Reply:
I don't know.  I didn't want to start changing things like that for now.